dnf-behave-tests/features/repoquery/main.feature

```diff
@@ -349,6 +349,29 @@ Given I successfully execute rpm with args "-i --nodeps {context.dnf.fixturesdir
       dummy-1:1.0-1.x86_64
       """
 
+   # --extras: installed pkgs, different NEVRA in available repository
+Scenario: dnf repoquery --extras (when there are such pkgs with different NEVRA in repository)
+ When I execute rpm with args "-i --nodeps {context.dnf.fixturesdir}/repos/dnf-ci-fedora/x86_64/wget-1.19.5-5.fc29.x86_64.rpm"
+ And I execute dnf with args "repoquery --installed"
+ Then stdout contains "wget-0:1.19.5-5.fc29.x86_64"
+ Given I use repository "dnf-ci-fedora-updates"
+ When I execute dnf with args "repoquery --extras"
+ Then the exit code is 0
+ Then stdout does not contain "wget-0:1.19.5-5.fc29.x86_64"
+ Then stdout does not contain "\*system\*"
```

Comment:
I think `repoquery` output has no variable parts and therefore you can check it with `stdout is ...`. Expecially in this case, when you do only a negative check, there could realistically be some erroneous output the negative check won't catch.

Reply:
I will improve it.